.pylintrc

```diff
@@ -67,6 +67,33 @@ forbidden-imports=
     core.domain:core.controllers,
     core.controllers:core.platform|core.storage
 
+disallowed-functions-and-replacements-str=now=>datetime.datetime.utcnow,
+assertEquals=>self.assertEqual,
+StringIO=>python_utils.string_io,
+urlparse.urlsplit=>python_utils.url_split,
+urlparse.urlparse=>python_utils.url_parse,
+urlparse.urlunsplit=>python_utils.url_unsplit,
+urlparse.parse_qs=>python_utils.parse_query_string,
+urlparse.urljoin=>python_utils.url_join,
+urllib.unquote=>python_utils.urllib_unquote,
+urllib.urlparse.unquote=>python_utils.urllib_unquote,
```

Comment:
Updated to use urlparse and urllib modules, since these are the functions whose implementations are being wrapped in `python_utils`